lib/conjur/conjur_config.rb

```diff
@@ -23,6 +23,7 @@ class ConjurConfig < Anyway::Config
     attr_config(
       # Read TRUSTED_PROXIES before default to maintain backwards compatibility
       trusted_proxies: (ENV['TRUSTED_PROXIES'] || []),
+      max_resources_limit: 0,
```

Comment:
What do you think about:
```
api_resource_list_limit_max
```
instead?

The thought process here is that each subsequent word in the name should provide a meaningful subdivision of the configuration space. This makes it easier to scan the file, and to potentially sort it alphabetically in a meaningful way.

`api` -> `resource` -> `list` -> `limit` -> `max`

Also interested in what others think from the @cyberark/conjur-core-team as we expand these configurations and identify some conventions for how that should be named? The goal is that they're consistent and predictable.

Reply:
Also, not sure if this is ready for final review or not, @aloncarmel111 , but this default value should include a comment explaining what the config setting is and what the default means. Thanks!

Reply:
Updated the parameter name to `api_resource_list_limit` and added comment.